### ISSUE TITLE
Validate `schema` for every operation and linked previous operations

### ISSUE THREAD

**Issue by adzialocha** [p2panda/aquadoggo#50]
We have to make sure that the `schema` field always stays the same per `document` for every new incoming entry. Also we have to check if all linked `previous_operations` are part of the same schema.

**Comment by adzialocha** [comment]
Related handbook issue: https://github.com/p2panda/handbook/issues/126

**Comment by sandreae** [comment]
We are now checking that every operation in `previous_operations` is part of the correct document, do we think this enough to also infer that it follows the correct schema? (I _think_ it is)

**Comment by adzialocha** [comment]
Yes, that's enough 👍 in the specification I'd still explicitly mention it.

**Comment by adzialocha** [comment]
> We are now checking that every operation in `previous_operations` is part of the correct document, do we think this enough to also infer that it follows the correct schema? (I _think_ it is)

So if an operation A follows correctly the schema S1 it will be stored. Now an operation B points at A but correctly follows schema S2 .. it would pass wrongly?
